gapis/resolve/dependencygraph2/README.md

```diff
@@ -0,0 +1,165 @@
+# Dependency graph
```

Comment:
It's probably easier to read here:

https://github.com/hevrard/agi/blob/171775891-doc-depgraph/gapis/resolve/dependencygraph2/README.md